packages/editor/src/components/widgets/tag/Tags.css

```diff
@@ -36,100 +36,3 @@
 .tag-add:focus-visible {
   box-shadow: var(--focus-shadow);
 }
-
```

Comment:
Not needed anymore. some dead bodies here?

Reply:
No, moved to Popover.css